### PR TITLE
295396 update install adapter section

### PR DESCRIPTION
**Pull request by osisoft-mbishop**

### CODE REVIEW

File: content/main/shared-content/installation/install-using-docker.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
+uid: InstallUsingDocker
+---
+
+# Installation using Docker
+
+Docker is a set of tools that you can use on Linux to manage application deployments. This topic provides examples of how to create a Docker container with the adapter.
+
+**Note:** If you want to use Docker, you must be familiar with the underlying technology and have determined that it is appropriate for your planned use of the adapter. Docker is not required to use the adapter.
```
Comment (KNyce71) [inline content/main/shared-content/installation/install-using-docker.md:9]:
```suggestion
**Note:** If you want to use Docker, OSIsoft recommends that you be both familiar with the underlying technology and have determined that it is appropriate for your planned use of the adapter. Docker is not required to use the adapter.
```

**Review by KNyce71 — COMMENTED** [review]
One minor suggestion identical to the one in PR #36.

File: content/main/shared-content/installation/install-using-docker.md (hunk 1 above)
Comment (rmuolic) [inline content/main/shared-content/installation/install-using-docker.md:9]:
```suggestion
**Note:** Installation using Docker is only recommended for users proficient with Docker. Docker is not required to use the adapter.
```

Comment (rmuolic) [inline content/main/shared-content/installation/install-using-docker.md:9]:
```suggestion
 **Note:** Installation using Docker is only recommended for users proficient with Docker. Docker is not required to use the adapter.
```

Comment (rmuolic) [inline content/main/shared-content/installation/install-using-docker.md:9]:
```suggestion
**Note:** The use of Docker is only recommended if your environment requires it. Only users proficient with Docker should use it to install the adapter. Docker is not required to use the adapter.
```

File: content/toc.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@
   items: 
   - topicUid: SystemRequirements
   - topicUid: InstallTheAdapter
-  - topicUid: InstallPIAdapterForDNP3UsingDocker
+  - topicUid: InstallUsingDocker
   - topicUid: UninstallTheAdapter
```
Comment (rmuolic) [inline content/toc.yml:10]:
```suggestion
  - topicUid: InstallUsingDocker
  - topicUid: UpgradeTheAdapter
  - topicUid: UninstallTheAdapter
```

**Review by rmuolic — CHANGES_REQUESTED** [review]
Minor changes.

File: content/main/shared-content/installation/install-using-docker.md (hunk 1 above)
Comment (osisoft-mbishop) [reply content/main/shared-content/installation/install-using-docker.md]: Rainier and I found a better way to word this section.

**Review by rmuolic — APPROVED** [review]

**Pull request merged**